.travis.yml

```diff
@@ -15,5 +15,5 @@ before_install:
   - tools/travis/nuke_pg.sh
 install:
   - tools/travis/install_pg.sh
-script: tools/travis/pg_travis_multi_test.sh
+script: CFLAGS=-Werror tools/travis/pg_travis_multi_test.sh
```

Comment:
@anarazel — I thought you were OK with this being set in Travis, where this is very controlled (Travis publicizes their GCC versions, we run on a known version of Ubuntu, etc.)

This is why I didn't do this in our code settings, but rather only in Travis (and only for this repo): it catches obvious mistakes where someone checks in a warning, but doesn't do the much-harder-to-control thing of setting `Werror` everywhere.